samples/typescript/kafka-trigger/README.md

```diff
@@ -0,0 +1,26 @@
+# Developers guid for Kafka Functions for TypeScript
+
+TypeScript is almost the same as [JavaScript](../../javascript/README.md). Please refer the [JavaScript](../../javascript/README.md). In the TypeScript sample, we don't provide confluent cloud sample. The configuration is the same as JavaScript. 
+
+## Samples
+
+We provide two samples.
+
+| Name | Description | Kafka Cluster| Enabled |
+| ----- | --------------- | -------| ---|
+| UsersTrigger | Simple Kafka trigger sample | local | yes |
+| UsersTriggerMany | Kafka batch processing sample | local | yes |
+
+## Difference between Javascript and TypeScript
+
+You need to compile your app before start Azure Functions Core Tools.
+
+```bash
+$ npm install -g typescript
```

Comment:
It looks like "typescript" is already included as a devDependency in the sample's package.json, so this shouldn't be needed. Should be an `npm install` and `npm run build`

Reply:
Done! Thanks!